### ISSUE TITLE
add og:Image

### ISSUE THREAD

**Issue by daknhh** [Drarig29/typedoc-plugin-extras#21]
I would be nice if you can add the posibility to set the og:Image tag. As a particularly click-attractive part of the Open Graph tags, og:image is very important. Choose an appealing image that encourages users to click or share. In order to be able to control the display in the stream of the target group as completely as possible, a specially created og:image is recommended, especially for socially relevant content. The article image, which is also used on the website itself, usually still has good optimization options. If you do not define the og:image tag on Facebook, for example, users can select an image themselves in the social preview. Since advertising banners, icons or other elements of the page are often also available for selection in this preview, defining the Open Graph image should be mandatory for you.

**Comment by Drarig29** [comment]
Looks like you used chatgpt to create the issue 😅

**Comment by daknhh** [comment]
@Drarig29 yes me and my friend wrote that together haha :P Maybe the og:title to be set as from the title tag and we could also set the og:description as the description?
